lib/schedule/trip.ex

```diff
@@ -130,8 +130,8 @@ defmodule Schedule.Trip do
   @doc """
   Whether the trip is active at any time during the time_of_day range.
   """
-  @spec is_active(t(), Util.Time.time_of_day(), Util.Time.time_of_day()) :: boolean()
-  def is_active(trip, start_time_of_day, end_time_of_day) do
+  @spec active?(t(), Util.Time.time_of_day(), Util.Time.time_of_day()) :: boolean()
+  def active?(trip, start_time_of_day, end_time_of_day) do
```

Comment:
I think this one might not be called anywhere - renaming it yielded no compilation errors.